lib/pymedphys/dicom.py

```diff
@@ -4,6 +4,7 @@
 # pylint: disable = unused-import
 
 from ._dicom.anonymise import anonymise_dataset as anonymise
+from ._dicom.connect.listen import DicomListener
```

Comment:
Hi @pchlap,

I actually missed that this was placed in the public API. That means that every attribute and method, and every function input variable name on that class that isn't proceeded with an `_` would become a part of PyMedPhys' public API. In the case of `DicomListener` that means all of the following functions and attributes can result in a breaking change if they need to be adjusted:

```python
~/git/pymedphys$ poetry run ipython
Python 3.8.6 (default, Nov  1 2020, 14:26:05) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from pymedphys.dicom import DicomListener                                                                                              

In [2]: dicom_listener = DicomListener()                                                                                                       

In [3]: [item for item in dir(dicom_listener) if not item.startswith('_')]                                                                     
Out[3]: 
['ae',
 'ae_title',
 'association_directory',
 'host',
 'on_association_accepted',
 'on_association_released',
 'on_c_echo',
 'on_c_store',
 'on_released_callback',
 'port',
 'start',
 'stop',
 'storage_directory']
```

Before we release version 0.34.0 would we be able to remove this line here from the exposed API? And then, in a future PR (doesn't have to be before 0.34.0) if we want to expose it via the API might we be able to expose it as a single function which internally boots up the class? That makes our API surface area significantly smaller and controllable.